--- Makefile.PL	Thu May 28 19:09:18 2009
+++ ../../../files/Makefile.PL.PerlMagick-mine	Thu Oct  1 23:43:08 2009
@@ -27,7 +27,7 @@
 # Compute test specification
 my $delegate_tests='t/*.t';
 my $delegate;
-foreach $delegate (qw/bzlib djvu fontconfig freetype jpeg jng jp2 lcms png tiff x11 xml wmf zlib/) {
+foreach $delegate (qw/bzlib fontconfig freetype jpeg jng png tiff x11 xml zlib/) {
   if ( -d "t/$delegate" ) {
     if ($delegate =~ /x11/) {
       if ( defined $ENV{'DISPLAY'} ) {
@@ -56,34 +56,34 @@
    'VERSION' => '6.5.2',
 
    # Preprocessor defines
-   'DEFINE'	=> ' -D_LARGE_FILES=1 -DHAVE_CONFIG_H',     # e.g., '-DHAVE_SOMETHING' 
+   'DEFINE'	=> ' -D_FILE_OFFSET_BITS=64 -D_LARGE_FILES=1 -DHAVE_CONFIG_H',     # e.g., '-DHAVE_SOMETHING' 
 
    # Header search specfication and preprocessor flags
-   'INC'	=> '-I../ -I.. -I/usr/include/freetype2 -I/usr/include/libxml2',
+   'INC'	=> '-I/opt/csw/include/ImageMagick -I/opt/csw/include -I/opt/csw/include/freetype2 -I/usr/sfw/include -I/usr/openwin/include -I/usr/include/libxml2',
 
    # C compiler
-   'CC' => 'gcc -std=gnu99',
+   'CC' => '/opt/SUNWspro/bin/cc -xc99=all',
 
    # C pre-processor flags (e.g. -I & -D options)
-   # 'CPPFLAGS' => "$Config{'cppflags'} -I/usr/include/freetype2 -I/usr/include/libxml2",
+   # 'CPPFLAGS' => "$Config{'cppflags'} -I/opt/csw/include -I/opt/csw/include/ImageMagick -I/opt/csw/include/freetype2 -I/usr/sfw/include -I/usr/openwin/include -I/usr/include/libxml2",
 
    # C compiler flags (e.g. -O -g)
-   'CCFLAGS' => "$Config{'ccflags'} -fopenmp -O3 -g -Wall -W -pthread",
+   'CCFLAGS' => "$Config{'ccflags'} -g -D_REENTRANT",
 
    # Linker
-   'LD' => $Config{'ld'} == $Config{'cc'} ? 'gcc -std=gnu99' : $Config{'ld'},
+   'LD' => $Config{'ld'} == $Config{'cc'} ? '/opt/SUNWspro/bin/cc -xc99=all' : $Config{'ld'},
 
    # Linker flags for building an executable
-   'LDFLAGS' =>  "-L../magick/.libs -lMagickCore -L../wand/.libs -lMagickWand $Config{'ldflags'} -Wl,--as-needed -lfreetype",
+   'LDFLAGS' =>  "-L/opt/csw/lib -R/opt/csw/lib -lMagickCore -lMagickWand $Config{'ldflags'} -Wl,-zlazyload -L/usr/openwin/lib -R/usr/openwin/lib -L/usr/sfw/lib -R/usr/sfw/lib -lfreetype",
 
    # Linker flags for building a dynamically loadable module
-   'LDDLFLAGS' => "-L../magick/.libs -lMagickCore -L../wand/.libs -lMagickWand $Config{'lddlflags'} -Wl,--as-needed -lfreetype",
+   'LDDLFLAGS' => "-L/opt/csw/lib -R/opt/csw/lib -lMagickCore -lMagickWand $Config{'lddlflags'} -Wl,-zlazyload -L/usr/openwin/lib -R/usr/openwin/lib -L/usr/sfw/lib -R/usr/sfw/lib -lfreetype",
 
    # Install PerlMagick binary into ImageMagick bin directory
-   'INSTALLBIN'	=> '/usr/bin',
+   'INSTALLBIN'	=> '/opt/csw/bin',
 
    # Library specification
-   'LIBS' => [ '-L../magick/.libs -lMagickCore -L../wand/.libs -lMagickWand -lperl -lm' ],
+   'LIBS' => [ '-L/opt/csw/lib -lMagickCore -lMagickWand -lperl -lm' ],
 
    # Perl binary name (if a Perl binary is built)
    'MAP_TARGET'	=> 'PerlMagick',
@@ -92,10 +92,10 @@
    # 'OPTIMIZE'	=> '',
 
    # Use same compiler as ImageMagick
-   'PERLMAINCC'	=> ' -fopenmp',
+   'PERLMAINCC'	=> '/opt/SUNWspro/bin/cc ',
 
    # Set Perl installation prefix to ImageMagick installation prefix
-#   'PREFIX'	=> '/usr',
+#   'PREFIX'	=> '/opt/csw',
 
    # Include delegate directories in tests
    test	=> { TESTS	=>	$delegate_tests},
@@ -117,6 +117,6 @@
      # tries to insert our library path first. Also, use the same compiler used
      # to build perlmain.c to link so that a C++ compiler may be used if
      # necessary.
-     $inherited =~ s:MAP_LINKCMD\s.*\s*\$\(CC\):MAP_LINKCMD   = \$(PERLMAINCC) -L/usr/lib: ;
+     $inherited =~ s:MAP_LINKCMD\s.*\s*\$\(CC\):MAP_LINKCMD   = \$(PERLMAINCC) -L/opt/csw/lib: ;
      $inherited;
  }
